hardware/AnnaThermostat.cpp

```diff
@@ -1,3 +1,5 @@
+#include <tinyxml.h>
```

Comment:
Could you move this line to the same old position ? (around line 11)
As you might know, it is needed to have stdafx.h as the first line
It is also a good idea to always include the class header as second (or in projects where no precompiled headers are in use, as first)